counter/src/index.js

```diff
@@ -4,7 +4,7 @@ import './index.css';
 import Counter from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<Counter />, document.getElementById('root'));
+ReactDOM.render(<Counter step = {1} init = {0} max = {5} message = {'Count is max'} />, document.getElementById('root'));
```

Comment:
Не оставляй такие длинные строки, их неудобно читать. Форматируй.
```suggestion
ReactDOM.render(
      <Counter 
           step = {1} 
           init = {0} 
           max = {5} 
           message = {'Count is max'}
       />, 
      document.getElementById('root')
);
```